scripts/sizeSnapshot/webpack.config.js

```diff
@@ -97,8 +97,9 @@ async function getSizeLimitBundles() {
 
 module.exports = getSizeLimitBundles;
 
-module.exports = new Promise(async resolve => {
-  const entry = (await getSizeLimitBundles()).reduce((acc, bundle) => {
+module.exports = async function webpackConfig() {
```

Comment:
Should've read the text instead of only the headline :laughing: 
https://webpack.js.org/configuration/configuration-types/#exporting-a-promise

Reply:
Throw an exception inside the method, it's swallowed.

Reply:
Well yes. That's the nature of unhandledPromiseRejections. Which is why I always do asyncFunction().catch()

Reply:
It's not enough.